### PR TITLE
Randy's HTML

### PR DESCRIPTION
**Pull request by RandyCastr0**

### CODE REVIEW

File: lesson_01/RandyCastro/index.html [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+</head>
+<body>
+
+<h1>Hello my name is Randy💫</h1>
+
+<p>Hello, I'm Randy Castro, and for the past six years, I've been deeply immersed in the dynamic world of retail management, specifically at Banana Republic. However I am now looking to transition into the tech industry. My ultimate goal is to keep on learning new things and branch out to multiple streams of income and retire early.</p>
+
+<h2>What's the best way to reach me? 📞</h2>
+
+<ol>
```
Comment (anthonydmays) [inline lesson_01/RandyCastro/index.html:17]: List indent is too far from the left margin, please adjust. Same for the list below.

File: lesson_01/RandyCastro/index.html [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+<p>I prefer to kickstart my day with a productive morning routine. Typically, I hit the gym around 7 am to get energized. Then, from 9 am to 5 pm, I focus on tackling my tasks and projects with full dedication and efficiency.</p>
+
+<h2>One of my goals for this year🏆</h2>
+
+<p>My goal is to successfully complete this program and secure a fulfilling career job in the tech industry. I aim to leverage this opportunity and eventually explore other avenues for personal and professional growth.</p>
+
+<h2>Family and Pets</h2>
+
+<p>In my household it is just my wife and our cat name Nori</p>
+
```
Comment (anthonydmays) [inline lesson_01/RandyCastro/index.html:42]: There are duplicated tags. There should only be one of each of the following tags: <html/>, <head/>, <body/>, and <title/>. Please adjust.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

File: lesson_01/RandyCastro/index.html
Comment (HeirJordan3) [inline lesson_01/RandyCastro/index.html:1]: Missing border under the titles.

**Review by HeirJordan3 — CHANGES_REQUESTED** [review]

**Review by anthonydmays — APPROVED** [review]
LGTM

**Pull request merged**